dotexporter.py

```diff
@@ -87,21 +102,102 @@ def do_GET(self):
         chain_getHeader = self.query("chain_getHeader")
         system_health   = self.query("system_health")
         runtime_version = self.query("state_getRuntimeVersion")
+        self.current_best = int(chain_getHeader['number'], 16)
+        best_drift=int(datetime.utcnow().timestamp()- self.previous['best_epoch'])
+
+
+        if self.current_best > self.previous['best']:
+           self.previous['best'] = self.current_best
+
+           if(datetime.utcnow().timestamp() >= self.previous['best_epoch']):
+              self.previous['best_epoch'] = int(datetime.utcnow().timestamp())
+              if DEBUG:
+                print("datetime.utcnow().timestamp()" + str(datetime.utcnow().timestamp()))
+           m.append({
+             'name': 'dot_stall_best',
+             'value': 0,
+              'prop': { 'block': self.current_best, 'drift': best _drift  }
```

Comment:
I would suggest not to use these as properties but as metrics instead. otherwise a new timeseries will be created when a new block is found. so the block is actually already a metric, maybe have another one for the drift.

Reply:
Resolved using output:

```
dot_chain_stall{name="parity-polkadot",version="0.7.16",chain="Development",block="best"} 0
dot_chain_stall{name="parity-polkadot",version="0.7.16",chain="Development",block="final"} 0
dot_chain_stall_drift{name="parity-polkadot",version="0.7.16",chain="Development",block="best"} 5
dot_chain_stall_drift{name="parity-polkadot",version="0.7.16",chain="Development",block="final"} 5
```